docs/BUILD

```diff
@@ -17,6 +17,7 @@ py_binary(
     ],
     data = [
         ":runfiles",
+        "//docs/codeexamples:code_files",
```

Comment:
I don't think we need this, let's have the `filegroup(name = 'docs')` defined in `plaidml/edsl/BUILD` include the `edsl_docs.cc` and `edsl_docs.py` files.